modules/nf-core/modules/ucsc/bedclip/main.nf

```diff
@@ -1,9 +1,8 @@
-def VERSION = '377' // Version information not provided by tool on CLI
-
 process UCSC_BEDCLIP {
     tag "$meta.id"
     label 'process_medium'
```

Comment:
the meta.yml corresponding to this module is empty, but I can see that we have one on modules, do you have a recent enough version of it?

Reply:
hmmm yes I updated all the modules. I will see after I do the lastest module update

Reply:
fixed